src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,94 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final int CAPACITY_MULTIPLIER = 2;
+    private static final float LOAD_FACTOR = 0.75f;
+    private int threshold;
+    private int size;
+    private Node<K, V>[] array;
+
+    public MyHashMap() {
+        this.threshold = (int) (INITIAL_CAPACITY * LOAD_FACTOR);
+        this.size = 0;
+        this.array = (Node<K, V>[]) new Node[INITIAL_CAPACITY];
```

Comment:
why are you casting?
```suggestion
        array = new Node[INITIAL_CAPACITY];
```